Mvc.sln

```diff
@@ -13,6 +13,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "test", "test", "{3BA657BF-2
 	ProjectSection(SolutionItems) = preProject
 		test\Directory.Build.props = test\Directory.Build.props
 		test\Directory.Build.targets = test\Directory.Build.targets
+		test\MvcTests.ruleset = test\MvcTests.ruleset
```

Comment:
Yes. Worth noting that its absence has no effect on the way it builds. It just harder to find via GUI.